test/extinction_test.jl

```diff
@@ -22,6 +19,9 @@ dprob = DiscreteProblem(u0,(0.,100.),rates)
 
 algs = DiffEqJump.JUMP_AGGREGATORS
 
+u0 = [10]
+dprob = DiscreteProblem(u0,(0.,10.),rates)
```

Comment:
Why did you decrease this to 10? We were getting occasional test failures at 10 before (rare events where it hadn't gone extinct by that time).